src/helpers/date-helpers.ts

```diff
@@ -15,6 +25,7 @@ export function getDatesForMonth(year: number, month: number) {
 	return monthDays;
 }
 
+// e.g., Monday or Friday, with special cases for today/yesterday
```

Comment:
Can these comments use the jsdoc style of commenting? This will allow the comment to be exported with the function.

Reply:
You mean just like:

```
/** 
 *  jsdoc comment here
 *  and here
 *  and even here
 */
 ```

Reply:
@lynnfaraday Yes, that's it